etc/config.json.coal

```diff
@@ -2,10 +2,7 @@
   "host": "0.0.0.0",
   "port": 4000,
   "sslport": 4443,
-  "ssl":{
-    "certificate":"/Users/kevin/joyent/sdc7/adminui/etc/ssl/default.pem",
-    "key":"/Users/kevin/joyent/sdc7/adminui/etc/ssl/default.pem"
-  },
```

Comment:
for development are u able to set up *without* "certificate" and "key" being defined? Otherwise, the file should continue to have placeholders here.